docs/source/keywords.rst

```diff
@@ -163,6 +163,14 @@ General options
       * Aggressively vectorize loops
       * ``OFF``
 
+    * * ``Kokkos_GLOBAL_COMPILATION``
+      * Set compiler to the CXX compiler Kokkos needs for the whole project
+      * ``ON``
+
+    * * ``Kokkos_FORCE_COMPILER_LAUNCHER``
+      * Use the kokkos_compiler_launcher script on all targets in the project. Redirects files in targets that link to Kokkos to the CXX compiler Kokkos needs.
+      * ``OFF``
```

Comment:
Aren't these more variables to set downstream when finding Kokkos?  (in which case they don't really belong to that file)

Reply:
done in https://github.com/kokkos/kokkos-core-wiki/pull/611/commits/5f78dc6330ee39b0f9a4f45bbb5f9c1090519bca